proto/buf.gen.python.yaml

```
@@ -4,3 +4,7 @@ managed:
 plugins:
   - name: python
     out: ../python
+
+  - plugin: protoc-gen-mypy
+    out: ../python
+    path: ../.venv/bin/protoc-gen-mypy
```

```suggestion
  - plugin: mypy
    out: ../python
```

bufのpluginは `plugin: name` で指定すると `proto-gen-{name}` をPATHから探すようなので、これで動くはずです。